tests/kafkatest/__init__.py

```diff
@@ -22,4 +22,4 @@
 # Instead, in development branches, the version should have a suffix of the form ".devN"
 #
 # For example, when Kafka is at version 1.0.0-SNAPSHOT, this should be something like "1.0.0.dev0"
-__version__ = '4.0.0.dev0'
+__version__ = '3.9.0.dev0'
```

Comment:
Hmm, should this be 3.10?

Reply:
This always confuses me. I never know which version should go in there.